cg/models/demultiplex/flow_cell.py

```diff
@@ -33,18 +33,18 @@
 class FlowCellDirectoryData:
     """Class to collect information about flow cell directories and their particular files."""
 
-    def __init__(self, flow_cell_path: Path, bcl_converter: Optional[str] = "bcl2fastq"):
+    def __init__(self, flow_cell_path: Path, bcl_converter: Optional[str] = None):
```

Comment:
I guess we should remove this argument then! Maybe better in a separate PR?

Reply:
See explanation above, tldr might be needed for reproducability for novaseq flowcells demultiplexed with bcl2fastq or hiseqx with bclconvert.